app/assets/javascripts/workflows.js.coffee

```diff
@@ -161,7 +161,7 @@ $(window).focus ->
     $("#terminal_button").bind('click', false)
 
 @update_submit_button = (id, status_char) ->
-  if id? && !status_char?
+  if id? && status_char != "R" && status_char != "Q"
```

Comment:
or better yet, don't duplicate the knowledge client side. There is already a method `@workflow.active?`. If that is not true then you should be able to resubmit it. So maybe we add an extra boolean attribute to each workflow in the JSON indicating whether its active or not, and use that, instead of checking the status_char.

Do we do this status_char checking anywhere else?